dashboard/app/helpers/users_helper.rb

```diff
@@ -62,18 +62,6 @@ def summarize_user_progress(script, user = current_user, exclude_level_progress
     user_data.compact
   end
 
-  # Get a users level progress within a certain script.
-  # Example return value (where 135 and 136 are levelIds):
-  #   {
-  #     "135": {"status": "perfect", "result": 100}
-  #.    "136": {"status": "perfect", "result": 100}
-  #   }
-  def user_progress_for_levels(script, user = current_user)
-    user_data = {}
-    merge_script_progress(user_data, user, script)
-    user_data[:levels]
-  end
```

Comment:
`UsersHelper` is included in at least 13 places (not including tests), but this utility method was very specific to this one API, so I moved it into api_controller.rb (and made changes).  This might violate fat-model-thin-controller but it feels better organized than the current state of affairs to me.